storage/diff_scheme/diff_manager.hpp

```diff
@@ -33,17 +40,35 @@ class Manager final
     virtual void OnDiffStatusReceived(Status const status) = 0;
   };
 
+  using OnDiffApplicationFinished =
+      std::function<void(generator::mwm_diff::DiffApplicationResult result)>;
```

Comment:
nit: i'd get rid of result in parentheses and leave it as ```std::function<void(generator::mwm_diff::DiffApplicationResult)>; ``` because it's as much clear without the param name as with this

Reply:
done